interface/wx/thread.h

```diff
@@ -503,10 +503,25 @@ class wxThreadHelper
 
         @since 2.9.2
 
-        @see OnDelete()
+        @see OnDelete(), OnExit()
     */
     virtual void OnKill();
 
+    /**
+        Callback called by Exit() before actually exiting the thread.
+        This function will not be called if the thread was @ref Kill() killed.
+
+        This function can be overridden by the derived class to perform some
+        specific task when the thread is exited. Notice that it will be
```

Comment:
I'd add that "the base class version does nothing and doesn't need to be called if this method is overridden.".